src/pages/NewSensor/NewSensorStepTwo.js

```diff
@@ -27,30 +28,32 @@ export const NewSensorStepTwoComponent = ({
 }) => (
   <TabContainer>
     <Paper height={320} headerText={vaccineStrings.new_sensor_step_two_title}>
-      <BreachConfigRow
-        type="HOT_CONSECUTIVE"
-        {...hotConsecutiveConfig}
-        updateDuration={updateDuration}
-        updateTemperature={updateTemperature}
-      />
-      <BreachConfigRow
-        type="COLD_CONSECUTIVE"
-        {...coldConsecutiveConfig}
-        updateDuration={updateDuration}
-        updateTemperature={updateTemperature}
-      />
-      <BreachConfigRow
-        type="HOT_CUMULATIVE"
-        {...hotCumulativeConfig}
-        updateDuration={updateDuration}
-        updateTemperature={updateTemperature}
-      />
-      <BreachConfigRow
-        type="COLD_CUMULATIVE"
-        {...coldCumulativeConfig}
-        updateDuration={updateDuration}
-        updateTemperature={updateTemperature}
-      />
+      <AfterInteractions placeholder={null}>
```

Comment:
the stepper for minutes on this page is funky - working on page 3 though

Reply:
Yeah, sorry! 
a537571

Done this here as a quick fix as I am actually changing the implementation of this state in a couple PRs down the line..